manifests/09_deployment.yaml

```diff
@@ -85,3 +85,16 @@ spec:
             requests:
               cpu: 10m
               memory: 20Mi
+        - name: vsphere-problem-detector
+          image: quay.io/openshift/origin-vsphere-problem-detector:latest
+          command:
+          - start
+          - --listen=0.0.0.0:6456
+          ports:
+          - containerPort: 6456
```

Comment:
can you use `8444`? The CSO container should have `8443` and I'd like to keep them close.